### PR TITLE
Prepare release 0.5.4

### PR DESCRIPTION
**Pull request by bonomat**
- first commit fixes the changelog from 0.5.3
- second commit includes a fix which is part of this PR: https://github.com/itchysats/itchysats/pull/2615/commitsSigned-off-by: Philipp Hoenisch <philipp@coblox.tech>
- third commit removes the dropdown menu

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## Unreleased
```
Comment (luckysori) [inline CHANGELOG.md]: 🔧 I don't think we want this change!

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [0.5.4] - 2022-08-05
```
Comment (luckysori) [inline CHANGELOG.md:10]: 🔧 We still want an empty `Unreleased` section on top of this.

**Review by luckysori — APPROVED** [review]
LGTM. Two minor fixes in the changelog.

**Review by da-kami — APPROVED** [review]

**Comment by luckysori** [comment]
I'm re-running the failed jobs because the CI failure comes from code that is not yet used. I'm working on a fix, but we shouldn't block the release on this.

**Comment by da-kami** [comment]
Note: We have to evaluate if we regard this as a blocker for the release: https://github.com/itchysats/itchysats/issues/2634
I am unsure. Will do a quick investigation to learn more about the CFD in question.

**Review by da-kami — CHANGES_REQUESTED** [review]
~waiting for a quick fix for https://github.com/itchysats/itchysats/issues/2634~

and fix for `acceptRollover failed - Accept failed: No active rollover` log spam, currently being tackled by @luckysori

**Review by da-kami — APPROVED** [review]
With merging https://github.com/itchysats/itchysats/pull/2649 and https://github.com/itchysats/itchysats/pull/2638 the release is ready now.

**Comment by Restioson** [comment]
When this lands, could we try disable `--verbose-spans` in the maker automation, unless there is still data we need to get on the ping peer timeouts? It will probably reduce the trace disk usage significantly

**Comment by da-kami** [comment]
> When this lands, could we try disable `--verbose-spans` in the maker automation, unless there is still data we need to get on the ping peer timeouts? It will probably reduce the trace disk usage significantly

To my knowledge there is nobody actively investigating `ping` timeouts at the moment, so I am happy to prepare a PR that removes `--verbose-spans` - thanks for the hint @Restioson :)

**Pull request merged**